cypress/support/commands.ts

```diff
@@ -396,6 +400,7 @@ declare global {
       ): Chainable;
       deleteEntitiesIfExists(name: string): Chainable<null>;
       clickSaveButton(): Chainable<null>;
+      openParagraphsModal(): Chainable<null>;
```

Comment:
Are you sure about the `Chainable` type? I would say that the return type is only `Chainable` in case that you return the `cy` object, but I could be wrong.

Reply:
I think it should be chainable. It also returns `cy.` which is a Cypress object?